Flask-Integration/spellCheck.py

```diff
@@ -60,6 +60,8 @@ def checkPhrase(phrase, lng):
     
     #print(textList)
     for word in textList: 
+        word = re.sub('^[^a-zA-Záéíóú]','',word) 
+        word = re.sub('[^a-zA-Záéíóú]*$','',word)
```

Comment:
Hi Manasa, 
Is there a reason the sub function is repeated?
Thanks,
Julian

Reply:
Hi Julian ,
I tried to use  or operator in sub function but  it didn't work accordingly. As a result, I had to perform it twice to remove special characters from the beginning and end.

Reply:
Perfect!
Thank you Manasan